### PR TITLE
[ZEPPELIN-5694] Support apache kyuubi

### PR DESCRIPTION
**Pull request by huage1994**
### What is this PR for?
Give an example to use Apache Kyuubi in JDBC interpreter.

### What type of PR is it?
Documentation

*Please leave your type of PR only*

### Todos
* [ ] - Task

### What is the Jira issue?
* Open an issue on Jira https://issues.apache.org/jira/browse/ZEPPELIN-5694

### How should this be tested?
* CI passed

### Questions:
* Does the licenses files need to update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? Yes

### CODE REVIEW

**Comment by zjffdu** [comment]
Thanks for the contribution @huage1994, is kerbose also supported? And it is better to give example how to enable kyuubi features via jdbc interpreter

**Comment by huage1994** [comment]
> And it is better to give example how to enable kyuubi features via jdbc interpreter

Thanks @zjffdu  a lot for suggestion !  I would improve it.

File: docs/interpreter/jdbc.md
Comment (yaooqinn) [inline docs/interpreter/jdbc.md:857]: cc @pan3793

Comment (pan3793) [reply docs/interpreter/jdbc.md:857]: We recommend using `org.apache.kyuubi:kyuubi-hive-jdbc-shaded:1.5.2-incubating` to connect Kyuubi.

https://kyuubi.readthedocs.io/en/latest/client/kyuubi_jdbc.html

File: docs/interpreter/jdbc.md
Comment (pan3793) [inline docs/interpreter/jdbc.md:841]: `org.apache.kyuubi.jdbc.KyuubiHiveDriver`

File: docs/interpreter/jdbc.md
Comment (pan3793) [inline docs/interpreter/jdbc.md:830]: Apache Kyuubi (Incubating)

**Comment by huage1994** [comment]
Hi all!  
I have updated the doc for the review of @pan3793 .

And  though we now use `KyuubiHiveDriver`,  I found the following code still relies on `HiveDriver`.
```
          if (jdbcURL != null && jdbcURL.startsWith("jdbc:hive2://")) {
            HiveUtils.startHiveMonitorThread(statement, context,
                    Boolean.parseBoolean(getProperty("hive.log.display", "true")), this);
          }
```

I would fix this problem.

**Review by jongyoul — DISMISSED** [review]

**Review by yaooqinn — DISMISSED** [review]

**Comment by huage1994** [comment]
> Thanks for the contribution @huage1994, is kerbose also supported? And it is better to give example how to enable kyuubi features via jdbc interpreter

Hi @zjffdu , kerberos is supported now.  Could you help review that?
Sorry for my late rely, 🤣 I met some problems with usage of Kerberos and all of them had been fixed.

**Comment by huage1994** [comment]
The CI is passed in my own forked repo.

**Comment by zjffdu** [comment]
Sorry for the late response @huage1994 , PR LGTM, trigger the CI again and wait for its result before merge

**Review by zjffdu — APPROVED** [review]

**Comment by huage1994** [comment]
> Sorry for the late response @huage1994 , PR LGTM, trigger the CI again and wait for its result before merge

Thanks!

**Review by jongyoul — APPROVED** [review]

**Comment by Ganlz** [comment]
Logs and result data are mixed in the query result. How to solve this problem?


20231205-152852(WeLinkPC)

**Pull request merged**
